modules/post/linux/gather/checkvm.rb

```diff
@@ -140,19 +139,37 @@ def run
       when /vmware virtual ide|vmware pvscsi|vmware virtual platform/i
         vm = "VMware"
       when /xen_mem|xen-vbd/i
-        vm =  "Xen"
+        vm = "Xen"
       when /qemu virtual cpu version/i
         vm = "Qemu/KVM"
       when /\/dev\/vmnet/
         vm = "VMware"
       end
     end
 
+    # Check for .dockerenv file
+    if not vm
```

Comment:
I recently disabled the `unless` warning in the MSF rubocop rules because it causes confusion. Leave it as-is if you wrote it as `if` originally.